app/src/styles/globals.css

```diff
@@ -111,3 +111,13 @@ iframe {
   width: 1em;
   height: 1em;
 }
+
+.verification-input-selected-char {
```

Comment:
shouldnt need if we remove this component... and if in the future you do need styling bes to use tailwind like rest of app

Reply:
Tailwind wasn't really up to the job here because of everything being buried in the verification input, so another great reason to ditch that.